src/components/message/message.stories.mdx

```diff
@@ -0,0 +1,140 @@
+import { useState } from 'react';
+import { Meta, Story, Preview, Props } from '@storybook/addon-docs/blocks';
+
+import Message from '.';
+import Button from '../button';
+
+<Meta
+  title="Message"
+  parameters={{ info: { disable: true }}}
+/>
+
+# Message
+A longer message which stays on screen to read and interpret.
+
+Presents a static message which stays on screen.
```

Comment:
We probably don't need both this sentence and the one above, either can be removed I think